### PR TITLE
Adds optional nameplate prop to ContentShare

### PR DESCRIPTION
**Pull request by Amethystix**
**Issue #:** 
#879 

**Description of changes:**
`VideoTile` allows for an optional nameplate prop. Have `ContentShare` accept this as well to add a nameplate to the screenshare.

**Testing**
1. Have you successfully run `npm run build:release` locally?
yes

3. How did you test these changes?
Passing this prop already worked before this change was made. Simply added the type to props as optional. I tested this using a local fork of the package.

4. If you made changes to the component library, have you provided corresponding documentation changes?
Yes

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Removed
 
 ### Changed
-
+- Updated `ContentShare` to have optional nameplate prop
```
Comment (xuesichao) [inline CHANGELOG.md:17]: nit: need a period at the end of sentence.

File: src/components/sdk/ContentShare/ContentShare.stories.mdx [hunk 2]
```diff
@@ -32,7 +32,7 @@ const App = () => {
 
   return (
     <MeetingProvider>
-      <ContentShare />
+      <ContentShare nameplate='Test name' />
```
Comment (xuesichao) [inline src/components/sdk/ContentShare/ContentShare.stories.mdx:35]: Could you change this to "Content share"

**Comment by xuesichao** [comment]
It seems our GitHub workflow does not support external contribution, because it fails to load AWS credential from GitHub VM for external PR. Would you mind if I cherry pick you commit and submit another PR?
@Amethystix

**Comment by Amethystix** [comment]
> It seems our GitHub workflow does not support external contribution, because it fails to load AWS credential from GitHub VM for external PR. Would you mind if I cherry pick you commit and submit another PR? @Amethystix

So long as it gets in, that would be fine by me; I didn't realize everyone contributing works at aws/amazon 😅 Guessing this will be a problem if I make other contributions going forward, by external I'm assuming you just mean non-amazon accounts.

Thanks for quick review, made changes! Appreciate it 😊

**Comment by devalevenkatesh** [comment]
> > It seems our GitHub workflow does not support external contribution, because it fails to load AWS credential from GitHub VM for external PR. Would you mind if I cherry pick you commit and submit another PR? @Amethystix
> 
> So long as it gets in, that would be fine by me; I didn't realize everyone contributing works at aws/amazon 😅 Guessing this will be a problem if I make other contributions going forward, by external I'm assuming you just mean non-amazon accounts.
> 
> Thanks for quick review, made changes! Appreciate it 😊

Yes, unfortunately, the PR checks that run use AWS credentials for which we use GitHub secrets. And those cannot be shared for external contributions. We understand that this is a pain currently, and we are eagerly awaiting feature from GitHub to allow private environment to access GitHub secrets securely to be accessed when PRs are from forks. But until then we are blocked on this.

**Comment by ltrung** [comment]
@Amethystix Thanks for your contribution. I will bring it up to the team to see if we could find a workaround so that PRs from forks could run integration tests.

**Pull request merged**